### PR TITLE
Remove unnecessary tokens

### PR DESCRIPTION
**Pull request by SaiSan-WMDE**
This gets rid of cursor tokens, which were storing default CSS properties.

Issue: [T268128](https://phabricator.wikimedia.org/T268128)

### CODE REVIEW

**Review by micgro42 — APPROVED** [review]
Looks good to me :)

**Review by jakobw — APPROVED** [review]

File: tokens/properties/components/Button.json [hunk 1]
```diff
@@ -336,7 +336,7 @@
                     "value": "{background.color.base.disabled.value}"
                 }
             }
-        },
+        },
```
Comment (micgro42) [inline tokens/properties/components/Button.json:339]: Accidental trailing white space :upside_down_face:

Comment (SaiSan-WMDE) [reply tokens/properties/components/Button.json:339]: woops

Comment (SaiSan-WMDE) [inline tokens/properties/components/Button.json:339]:
```suggestion
        },
```

**Review by micgro42 — APPROVED** [review]

**Pull request merged**